### PR TITLE
magento/pwa-studio#2253: CREATE ACCOUNT. Mutation does not send a subscribe flag

### LINKED ISSUE #2253: [bug]: CREATE ACCOUNT. Mutation does not send a subscribe flag
**Describe the bug**

**To reproduce**
1. Go to 'Create Account'
2. Click on 'Subscribe to news and updates'
3. Click 'Submit'

**Expected behavior**
Argument [is_subscribed](https://devdocs.magento.com/guides/v2.3/graphql/mutations/create-customer.html) should be send.

**Screenshots**

**Additional context**

**Possible solutions**
Add a missing field into [useCreateAccount.js](https://github.com/magento/pwa-studio/blob/develop/packages/peregrine/lib/talons/CreateAccount/useCreateAccount.js#L74)

**Please complete the following device information:**
 - Device [e.g. iPhone6, PC, Mac, Pixel3]:
 - OS [e.g. iOS8.1, Windows 10]:
 - Browser [e.g. Chrome, Safari]:
 - Browser Version [e.g. 22]:
 - Magento Version:
 - PWA Studio Version:
 - NPM version `npm -v`:
 - Node Version `node -v`:

**Please let us know what packages this bug is in regards to:**
- [ ] `venia-concept`
- [ ] `venia-ui`
- [ ] `pwa-buildpack`
- [X] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`
- [ ] `create-pwa`

### PR DESCRIPTION
**Pull request by atwixfirster**
## Description

PR fixes [[bug]: CREATE ACCOUNT. Mutation does not send a subscribe flag](https://github.com/magento/pwa-studio/issues/2253)

## Related Issue

Closes #2253.

### Verification Steps

1. Go to 'Create Account'
2. Click on 'Subscribe to news and updates'
3. Click 'Submit'

**Expected result:**

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.

### CODE REVIEW

**Comment by atwixfirster** [comment]
@dani97 , could you please 
- approve PR again 
- and add `A version label is required. A maintainer must add one.`

Thank you!

**Comment by dani97** [comment]
> @dani97 , could you please
> 
>     * approve PR again
> 
>     * and add `A version label is required. A maintainer must add one.`
> 
> 
> Thank you!

@atwixfirster :+1: , Can you ask pwa team to merge this in slack channel. Also check with them why one of the tests are failing.

File: packages/peregrine/lib/talons/CreateAccount/useCreateAccount.js [hunk 1]
```diff
@@ -74,7 +74,8 @@ export const useCreateAccount = props => {
                         email: formValues.customer.email,
                         firstname: formValues.customer.firstname,
                         lastname: formValues.customer.lastname,
-                        password: formValues.password
+                        password: formValues.password,
+                        is_subscribed: typeof formValues.subscribe === 'boolean' ? formValues.subscribe : false
```
Comment (sirugh) [inline packages/peregrine/lib/talons/CreateAccount/useCreateAccount.js:78]: Can we just use `!!formValues.subscribe`?

Comment (atwixfirster) [reply packages/peregrine/lib/talons/CreateAccount/useCreateAccount.js:78]: I think we can everything. Will fix soon. 

Thank you, @sirugh

Comment (atwixfirster) [reply packages/peregrine/lib/talons/CreateAccount/useCreateAccount.js]: **done**

@sirugh , could you please approve?

Thank you!

**Pull request merged**